docs/decisions/0044-OTel-semantic-convention.md

```diff
@@ -0,0 +1,326 @@
+﻿---
+# These are optional elements. Feel free to remove any of them.
+status: { proposed }
+contact: { Tao Chen }
+date: { 2024-04-15 }
+deciders: {}
+consulted: { Ben Thomas, Liudmila Molkova, Stephen Toub }
+informed: { Dmytro Struk, Mark Wallace }
+---
+
+# Use standardized vocabulary and specification for observability in Semantic Kernel
+
+## Context and Problem Statement
+
+Observing LLM applications has been a huge ask from customers and the community. This work aims to ensure that SK provides the best developer experience while complying with the industry standards for observability in generative-AI-based applications.
+
+For more information, please refer to this issue: https://github.com/open-telemetry/semantic-conventions/issues/327
+
+### Semantic conventions
+
+The semantic conventions for generative AI are currently in their nascent stage, and as a result, many of the requirements outlined here may undergo changes in the future. Consequently, several features derived from this Architectural Decision Record (ADR) may be considered experimental. It is essential to remain adaptable and responsive to evolving industry standards to ensure the continuous improvement of our system's performance and reliability.
+
+- [Semantic conventions for generative AI](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/gen-ai)
+- [Generic LLM attributes](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/llm.md)
+
+### Telemetry requirements (Experimental)
+
+Based on the initial version, Semantic Kernel should provide the following attributes in activities that represent individual LLM requests:
```

Comment:
```suggestion
Based on the [initial version](https://github.com/open-telemetry/semantic-conventions/blob/651d779183ecc7c2f8cfa90bf94e105f7b9d3f5a/docs/attributes-registry/gen-ai.md?plain=1#L1), Semantic Kernel should provide the following attributes in activities that represent individual LLM requests:
```

here permalink seems to be more relevant. 

As you know some updates/changes are coming soon.